ktor-client/ktor-client-tests/build.gradle.kts

```diff
@@ -91,6 +91,7 @@ kotlin.sourceSets {
     val jvmTest by getting {
         dependencies {
             api(project(":ktor-client:ktor-client-apache"))
+            api(project(":ktor-shared:ktor-serialization:ktor-serialization-gson"))
```

Comment:
There is already `kotlinx-serialization-json` dependency in common. Can you use it?

Reply:
Yes, removed `ktor-serialization-gson` dependency